src/components/auth/signIn/email/SignInModalContent.module.sass

```diff
@@ -0,0 +1,101 @@
+@import 'src/styles/subsocial-vars.scss'
+
+.SignInModalContent
+    display: flex
+    flex-direction: column
+    justify-content: center
+    padding: 0 $space_tiny
+    gap: $space_normal
+    margin-top: $space_big
+
+.ButtonLinkDiv
+    font-size: $font_normal
+    font-weight: $font_weight_semibold
+    margin-left: $space_mini
+    padding: 0px
+
+.ButtonLinkMedium
+    font-size: $font_normal
+    font-weight: $font_weight_medium
+    &:disabled
+        color: #888888
```

Comment:
lets use vars

Reply:
Applied in [21ffc94](https://github.com/dappforce/polkaverse/pull/74/commits/21ffc947098855e4f1a52b34e2f240282e4ee4da)